tests/templates/test_layers.py

```diff
@@ -319,11 +319,11 @@ def test_different_circuits_for_different_seeds(self, tol):
         weights = [[0.1, 0.2, 0.3]]
 
         def circuit1(weights):
-            RandomLayers(weights=weights, wires=range(2), seed=1)
+            RandomLayers(weights=weights, wires=range(2), seed=10)
```

Comment:
:thinking:

Reply:
Ehm ja. I hoped you'd overlook that. It is one of those randomness tests where we use a seed, but some seeds (by sheer statistics) lead to a failure of the test, because the same architecture gets chosen.

I could also increase the wires or layers here, to make this less likely, but another seed does the same job.  Shall I rather rewrite?

Reply:
Oh, I see! Makes sense. No, honestly I think it's a valid idea to test that indeed there exist different seeds for which two different circuits can be obtained by `RandomLayers`. Not entirely sure how this could be made non-stochastic (and would also be out of scope for this PR). I assume that this test case failed this time due to the change in the `random_layer` function, correct?

Reply:
Yes!